src/ZkFold/Cardano/Plonk/OnChain/Data.hs

```diff
@@ -1,12 +1,11 @@
-{-# LANGUAGE TemplateHaskell #-}
 {-# LANGUAGE DeriveAnyClass  #-}
+{-# LANGUAGE TemplateHaskell #-}
 -- {-# LANGUAGE DeriveGeneric   #-}
```

Comment:
Maybe remove this line?

Reply:
TemplateHaskell is used in the file, but I can remove the commented line